### ISSUE TITLE
Parse latex string gives wrong result

### ISSUE THREAD

**Issue by DylanXie123** [viktorstrate/algebra-latex#12]
Parse expression '3/4/5' gives wrong result, it should be '3/20', but it gives '15/4' instead.
```js
const algebraLatex = require("algebra-latex");
const algebrite = require('algebrite');

const parser = new algebraLatex();
let expStr = '3/4/5';
parser.parseLatex(expStr);
let exp = parser.toAlgebra(algebrite);
console.log(exp.toString()) // gives "15/4" here
```

**Comment by viktorstrate** [comment]
This should be fixed in v2.0.1

**Comment by DylanXie123** [comment]
That works, thanks!
